packages/web-components/src/components/table-of-contents/table-of-contents.ts

```diff
@@ -306,8 +306,20 @@ class DDSTableOfContents extends HostListenerMixin(StableSelectorMixin(LitElemen
    */
   private _handleUserInitiatedJump(target: string) {
     const elem = this.querySelector(`a[name="${target}"]`);
-    elem?.scrollIntoView();
-    if (elem) {
+    const masthead = this.ownerDocument.querySelector('dds-masthead');
```

Comment:
Thanks for this PR!! Looks great. Just a 1 minor comment besides changing the prefix. Instead of using `48` we need to account for the masthead L1 being visible as well, so possibly creating a `mastheadHeight` variable and using `masthead` to get the `offsetHeight` to replace the `48`. 

```suggestion
    const masthead = this.ownerDocument.querySelector('${ddsPrefix}-masthead');
```

Reply:
This looks great!! Thanks for the updates :)